doc/version_history.rst

```diff
@@ -2,6 +2,13 @@
 Version History
 ===============
 
+v1.5.12
+-------
+
+* Improve the unit test.
```

Comment:
Please add towncrier files for this and remove the version history changes. The idea of towncrier is to have that update the version history when the release is made.